src/App.tsx

```diff
@@ -17,18 +78,35 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                onChangeQuery={setQuery}
+                onChangeTypeOfLoad={setTypeOfLoad}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading && !isError
+                ? (
```

Comment:
Looks better in my opinion 
```suggestion
              {isLoading && !isError ? (
```